DevCGlobalDirectory.md

```diff
@@ -365,6 +365,19 @@
 </details>
 
 
+<details>
+  <summary>
+    <b>DevC Nairobi</b>
+  </summary>
+
+  - Lead(s):
+    - [Eddie Kago](https://github.com/scripttie)
+    - [Carol Kariuki]
```

Comment:
**Issue:** This is incorrect MarkDown. Perhaps ping Carol on FB and ask her to share a link she's comfortable with. Wdyt?

Reply:
Her Github link?